src/components/Sum.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Sum = (props) => {
```

Comment:
ПРиучайся сразу делать деструктуризацию props 
```suggestion
const Sum = ({ a, b }) => {
```